### ISSUE TITLE
Create/Vote On Group Chat? (Vote In Comments)

### ISSUE THREAD

**Issue by NaomiHoniker** [CSI-280/MusicGames#5]
We need some sort of way to communicate quickly with one another. I was thinking maybe we set up something faster than email/leaving Github comments. We could vote in the comments here on what we do?

Options I can think of off the top of my head, but feel free to suggest more below:
1) Discord (our previous team set up a bot that notified us of Github updates)
2) Facebook Messager
3) Telegram (idk if people actually use Telegram, I just like it, personally)
4) Everyone promises to constantly check Github Issues and Comments (don't recommend this one, I just wanted a fourth option)
5) There is already a group chat and I don't know about it and my issue is irrelevant oops

[![](https://api.gh-polls.com/poll/01E2K2YB54YCSFT9234SSRNAKE/Discord)](https://api.gh-polls.com/poll/01E2K2YB54YCSFT9234SSRNAKE/Discord/vote)
[![](https://api.gh-polls.com/poll/01E2K2YB54YCSFT9234SSRNAKE/Messenger)](https://api.gh-polls.com/poll/01E2K2YB54YCSFT9234SSRNAKE/Messenger/vote)
[![](https://api.gh-polls.com/poll/01E2K2YB54YCSFT9234SSRNAKE/Telegram)](https://api.gh-polls.com/poll/01E2K2YB54YCSFT9234SSRNAKE/Telegram/vote)
[![](https://api.gh-polls.com/poll/01E2K2YB54YCSFT9234SSRNAKE/Github%20only)](https://api.gh-polls.com/poll/01E2K2YB54YCSFT9234SSRNAKE/Github%20only/vote)

**Comment by NaomiHoniker** [comment]
Personally, my vote would go to Discord, but I understand that not everybody necessarily has a Discord account. While it is free, if people really don't want to make one, we can just use Facebook Messager.

**Comment by chrisbendel** [comment]
if everyone had an iphone you could use imessage, #iphonemasterrace

**Comment by chrisbendel** [comment]
@CSI-280/web-people place your votes here

**Comment by A-Waters** [comment]
I'm not sure exactly how to "vote" but I'm down for discord
also Iphone = bad, so no imessage

**Comment by jbuzzell** [comment]
i am biased against discord personally so my vote is gonna go towards messenger (do people still primarily use messenger? am i a dinosaur?) if we were to use something similar to discord i would rather use slack.

**Comment by chrisbendel** [comment]
messenger is for boomers

**Comment by jbuzzell** [comment]
blocked

**Comment by lenora4321** [comment]
I do not have iphone and I do not want to use discord, I have it muted for various reasons

**Comment by chrisbendel** [comment]
you guys can have an email chain and go OG linux style as long as people actually check emails

**Comment by NathanDiMauro** [comment]
I though the point of this class was to learn to use git and github. I feel that messaging and communicating through github is the main point. It's kind of supposed to be a cluster f*ck.

**Comment by jbuzzell** [comment]
@NathanDiMauro i don't necessarily disagree but the point is more to manage the clusterfuck/minimize distress in said situations--organizing communication is an important part of that. that said, pretty much any form of communication online has its pros and cons, it's mostly about finding the one that works best for _us_

**Comment by chrisbendel** [comment]
I agree with you guys, the point is for it to be a clusterfuck - but the goal (learning objective) of the course is managing the cluster fuck in a way that works best for each individual team. Like I always say in class, you're going to end up on a team at a company with any possible dynamic, and you have to adapt to what works best for the (majority of) the team

That being said I think this issue is a perfect example of learning how to manage that clusterfuck, everyone is communicating and weighing in with their opinions, which honestly is awesome and a good exercise for everyone to be a part of.

TLDR Keep having discussions for the rest of the semester like this, and you will be learning important skills. Pick a communication medium (or mediums plural) that works best for most if not all of the team (you cant always make everyone happy)

**Comment by A-Waters** [comment]
can we close this?

**Comment by NaomiHoniker** [comment]
> can we close this?

Sure, I'll close it up for now, and if we decide that we don't like github communicating, then we'll open it back up and poll again. Personally, I think that a chat of some kind is just generally going to be helpful, but if nobody wants to do that, then I guess that's just the way it is. Maybe those of us who do want a chat can just make one together and fill people in that aren't in it? I dunno. Anyways, point is, yes. I'll close this.
